docs/modules/publisher-migrationpath.md

```diff
@@ -55,6 +55,78 @@ Please consider migrating your old published nodes JSON files that use `NodeId`
 
 To learn more about how to use comman-line arguments to configure OPC Publisher, please refer to [this](publisher-commandline.md) doc.
 
+## OPC Publisher 2.5.x Command Line Arguments supported in 2.8.2
+
+The following table describes the command line arguments, which were available in OPC Publisher 2.5.x and their compatibility in OPC Publisher 2.8.2.
```

Comment:
Can you please add a link to `publisher-commandline.md` saying that the full list of command line options for both versions is available there ?

Reply:
The link is already present in the paragraph above "Command Line Arguments", It's just a couple of lines above

Reply:
Ah, great. Didn't see it. Thanks.